service/initialise.go

```diff
@@ -94,7 +94,19 @@ func (e *ExternalServiceList) GetKafkaConsumer(ctx context.Context, cfg *config.
 // DoGetKafkaConsumer returns a Kafka Consumer group
 func (e *Init) DoGetKafkaConsumer(ctx context.Context, cfg *config.Config) (dpkafka.IConsumerGroup, error) {
 	cgChannels := dpkafka.CreateConsumerGroupChannels(1)
-	kafkaOffset := dpkafka.OffsetNewest
+
+	pConfig := &dpkafka.ProducerConfig{
+		KafkaVersion: &cfg.KafkaVersion,
+	}
+	if cfg.KafkaSecProtocol == "TLS" {
```

Comment:
The value "TLS" should be a constant which should be ideally mentioned in the `config.go`. Use this as an example: https://github.com/ONSdigital/dp-observation-extractor/blob/develop/config/config.go#L13